sql/core/src/main/scala/org/apache/spark/sql/artifact/ArtifactManager.scala

```diff
@@ -465,7 +465,7 @@ object ArtifactManager extends Logging {
   val ARTIFACT_DIRECTORY_PREFIX = "artifacts"
 
   private[artifact] lazy val artifactRootDirectory =
-    Utils.createTempDir(ARTIFACT_DIRECTORY_PREFIX).toPath
+    Utils.createTempDir(namePrefix = ARTIFACT_DIRECTORY_PREFIX).toPath
```

Comment:
I agree with @cxzl25 that this is a bug fix. Since this is an obvious bug fix, let me merge this to master/4.0.